www/source/blog/2019-01-04-0.72.0-release.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Habitat 0.72.0 Released
+date: 2019-01-04
+author: Matt Wrock
+tags: release notes
+category: product
+classes: body-article
+---
+
+Habitat 0.72.0 Release Notes
+
+We are happy to announce the release of Habitat v0.72.0. We have a new features, as well
+as bug fixes so please read on for all the details. If you just want the binaries, head on over to
+[Install Habitat](https://www.habitat.sh/docs/using-habitat/#install-habitat).
+
+As always thank you for using Habitat!
+
+Higlights:
```

Comment:
s/Higlights/Highlights

Reply:
HA! Nice catch. Since these are mostly copy/paste, that has been around for a few releases at least.